ui/static/ui/js/learning_resources.jsx

```diff
@@ -378,9 +399,13 @@ define('learning_resources',
         thiz.setState({
           message: "Form saved successfully!"
         });
+
+        // Exit panel
         if (closePanel) {
           thiz.props.closeLearningResourcePanel();
         }
+
+        thiz.props.markDirty(false);
```

Comment:
Can you move this inside `closeLearningResourcePanel()`? Once we decide to close the panel it would make sense to always clear the dirty flag there